Add Time Planner

Jamalianpour opened the pull request:
A widget to show tasks on customizable calendar

renancaraujo commented:
Hello there 👋 

Thanks for the submission and effort on the package that surely will help the open-source community.
I have some concerns tho:
- Any plans on supporting non-nullability?
- Will this lib be constantly maintained and updated?
- Is this package used somewhere? I see 0% populararity on pub.
- Any plans on adding tests?

Jamalianpour commented:
Hello renancaraujo,

> Will this lib be constantly maintained and updated?

Yes of course, this lib will be maintained and updated and response to any issue about that.

> Is this package used somewhere? I see 0% populararity on pub.

This package is a part of my own project and after that I decide to publish it as open-source and it uploaded to pub yesterday, that's why 0% popularity.

> Any plans on supporting non-nullability?
> Any plans on adding tests?

As soon as possible I'll add test and null safety to this.

Solido commented:
Hi !
To my knowledge nothing similar so this can get in.
Thx for PR @Jamalianpour

Jamalianpour commented:
Hi @Solido 
Now it support Null Safety.
please merge this pull request.
